nav2_recoveries/plugins/back_up.cpp

```diff
@@ -16,25 +16,29 @@
 #include <ctime>
 #include <memory>
 
-#include "nav2_recoveries/back_up.hpp"
+#include "back_up.hpp"
 
 using namespace std::chrono_literals;
 
 namespace nav2_recoveries
 {
 
-BackUp::BackUp(rclcpp::Node::SharedPtr & node, std::shared_ptr<tf2_ros::Buffer> tf)
-: Recovery<BackUpAction>(node, "BackUp", tf)
+BackUp::BackUp()
+: Recovery<BackUpAction>()
 {
   simulate_ahead_time_ = 2.0;
-  node_->declare_parameter("simulate_ahead_time");
-  node_->get_parameter("simulate_ahead_time", simulate_ahead_time_);
 }
 
 BackUp::~BackUp()
 {
 }
 
+void BackUp::onConfigure()
+{
+  node_->declare_parameter("simulate_ahead_time");
```

Comment:
Why don't we set the default here in the `declare_parameter` instead of in the constructor?

Reply:
No particular reason, I can update that

Reply:
Changes have been reflected